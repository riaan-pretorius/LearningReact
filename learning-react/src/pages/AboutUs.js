import React, { Component } from 'react';
import { Container, Row, Col } from 'reactstrap'

class AboutUs extends Component {
  render() {
    return (
        <Col>
            <h1>About Us</h1>
            <p>We got nothing</p>
        </Col>
    );
  }
}
const styles = {
  
}

export default AboutUs;
