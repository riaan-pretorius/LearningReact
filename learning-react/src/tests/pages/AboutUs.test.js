import React from 'react'
import AboutUs from '../../Pages/AboutUs'
import {mount, shallow} from 'enzyme';
import renderer from 'react-test-renderer'

test('AboutUs Test Snapshot', () => {
  const component = renderer.create(
    <AboutUs />
  )
  let test = component.toJSON();
  expect(test).toMatchSnapshot();
});

test('AboutUs Test', () => {
  const page = mount(
    <AboutUs />
  )
  
  let header = page.find('h1')
  expect(header.exists()).toBeTruthy()
  expect(header.text()).toBe("About Us")

  let p = page.find('p')
  expect(p.exists()).toBeTruthy()
  expect(p.text()).toBe("We got nothing")
})
