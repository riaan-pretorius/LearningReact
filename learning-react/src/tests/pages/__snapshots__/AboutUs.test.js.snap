// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs Test Snapshot 1`] = `
<div
  className="col"
>
  <h1>
    About Us
  </h1>
  <p>
    We got nothing
  </p>
</div>
`;
